src/Plugin.tsx

```diff
@@ -6,13 +6,17 @@ import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
 import { WidgetCollapsible } from './Components/WidgetCollapsible';
 import { GrowthChart } from './Components/GrowthChart/GrowthChart';
 import { EnrollmentOverviewProps } from './Plugin.types';
+import { useTeiById } from './utils/DataFetching/Hooks';
+import { useChartConfig } from './utils/DataFetching/Hooks/useChartConfig';
 
 const queryClient = new QueryClient();
 
 const PluginInner = (propsFromParent: EnrollmentOverviewProps) => {
-    const [open, setOpen] = useState(true);
-
+    const { chartConfig } = useChartConfig();
     const { teiId } = propsFromParent;
```

Comment:
why is this propsFromParent? just curious

Reply:
This is how we can get props form the url when using a plugin in the Capture app